hoodie-spark/src/main/java/com/uber/hoodie/BaseAvroPayload.java

```diff
@@ -29,9 +32,14 @@
 
 
   /**
-   * Avro data extracted from the source
+   * Avro data extracted from the source converted to bytes
    */
-  protected final GenericRecord record;
+  protected final byte [] recordBytes;
```

Comment:
This means, there is no need to register the schemas in HoodieDeltaStreamer & Spark Source anymore? 

@bvaradar can we make a call here?